pom.xml

```diff
@@ -2,14 +2,15 @@
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
 
-    <groupId>com.github.batkinson.plugins</groupId>
+    <groupId>com.github.pixeon.plugins</groupId>
     <artifactId>maven-pack200-plugin</artifactId>
-    <version>1.1-SNAPSHOT</version>
+    <version>1.1.0</version>

```

Comment:
Same here, a proper pull request shouldn't assume that you are dictating the release as part of it.